RELEASE.md

```diff
@@ -1,3 +1,18 @@
+# Release 2.11.0
+
+The 2.11 minor series tracks TensorFlow 2.11.
+
+## Bug Fixes
+
+- Fixed HParams plugin sorting bugs (#5925)
```

Comment:
I think this is the wrong PR - it should actually be go/tbpr/5971 I believe?

Reply:
Whoops. Must have been a copy paste error. Good catch. Fixed.